syntax = "proto3";  // Usamos la sintaxis de gRPC versi칩n 3

// Definici칩n del servicio
service MyService {
  // Definici칩n del RPC SayHello
  rpc SayHello (HelloRequest) returns (HelloResponse);
}

// Mensaje para la solicitud
message HelloRequest {
  string name = 1;  // El mensaje recibir치 un campo 'name' de tipo string
}

// Mensaje para la respuesta
message HelloResponse {
  string message = 1;  // El mensaje de respuesta contiene un campo 'message' de tipo string
}
